contracts/test/testContracts/ERC677ReceiverTest.sol

```diff
@@ -10,6 +10,8 @@ contract ERC677ReceiverTest is ERC677Receiver {
     uint public someVar = 0;
 
     function onTokenTransfer(address _from, uint256 _value, bytes _data) external returns(bool) {
+        require(_value != 666, "Very 6 testing");
```

Comment:
Maybe we can use a contract variable to decide if the next onTokenTransfer should success or not, instead of setting a magic number here.

Reply:
it make sense! added `forceCallbackFailure` method